src/main/kotlin/org/stellar/walletsdk/util/FetchAccountFromAddress.kt

```diff
@@ -0,0 +1,13 @@
+package org.stellar.walletsdk.util
+
+import java.io.IOException
+import org.stellar.sdk.Server
+import org.stellar.sdk.responses.AccountResponse
+
+fun fetchAccountFromAddress(accountAddress: String, server: Server): AccountResponse {
+  try {
+    return server.accounts().account(accountAddress)
+  } catch (e: IOException) {
```

Comment:
Isn't there a `NotFound` exception or something similar? `IOException` will raise an error if it cannot connect to horizon, but currently the SDK will inform the account wasn't found, which is inaccurate.

Reply:
We'll create proper exceptions once the basic flow is working and we know exactly what kind of exceptions we need.